mi/reporting/paths.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+
+_PATH_TO_HERE = Path(__file__).parent
+
+PATH_TO_QUERIES = _PATH_TO_HERE / "queries"
```

Comment:
There are a lot of global variables in a lot of files, I dont know if we want to have a more central place for them within the context of the MI Folder to make them a bit easier to locate? just a suggestion